evaluation/WhiteNoise-Core Evaluation.ipynb

```diff
@@ -0,0 +1,327 @@
+{
+ "cells": [
```

Comment:
can we take the space out of the name. We should find a convention but spaces are hard

Reply:
Sure. Fixed the name removing space